lamindb_setup/_migrate.py

```diff
@@ -85,7 +85,6 @@ def create(cls) -> None:
         setup_django(settings.instance, create_migrations=True)
 
     @classmethod
-    @disable_auto_connect
```

Comment:
I don't know why we had this here. With the new setup it understandably breaks `lamin migrate deploy`. I don't think we need it so I'm removing it.